CHANGELOG.md

```
@@ -30,7 +30,7 @@ DetailVariableDef.ViewGenerator = null; // or resolving of interface IViewGenera
 8. Constructor of class `AuditService`.
 9. Constructor of class `BusinessServerProvider`.
 10. Version of `Microsoft.Data.SqlClient` changed to `6.1.3`.
-11. Version of `Npgsql` changed to `4.0.17` for `.NETFramework4.5` and to `5.0.18` for `.NETStandard2.0` and upper.
+11. Version of `Npgsql` changed to `4.0.17` for `.NETFramework4.5`, to `5.0.18` for `.NETStandard2.0` and upper, to `10.0.0 for `.NET10`.
```

markdownlint-cli2 (0.18.1)

Spaces inside code span elements

(MD038, no-space-in-code)